Source/Extensions/Blazorise.Markdown/wwwroot/markdown.js

```diff
@@ -1,3 +1,8 @@
+import "./vendors/easymde.js?v=1.1.0.0";
+import "./vendors/highlight.js?v=1.1.0.0";
+
+document.getElementsByTagName("head")[0].insertAdjacentHTML("beforeend", "<link rel=\"stylesheet\" href=\"_content/Blazorise.Markdown/vendors/easymde.css?v=1.1.0.0\" />");
```

Comment:
Altough it's automatic, users loose the flexibility of configuring the link however they want. Load first, in last, with some certain setting, etc? Maybe it could inspect the head verifying if it already exists or not?

Reply:
Oh I was talking about the link bit. But yea it'd make sense if you could update all conditionally. I don't think it's possible to import modules conditionally either.